site/src/pages/CreateWorkspacePage/CreateWorkspacePage.test.tsx

```diff
@@ -201,25 +201,17 @@ describe("CreateWorkspacePage", () => {
     );
   });
 
-  it("external auth: errors if unauthenticated and submits", async () => {
+  it("external auth: errors if unauthenticated", async () => {
     jest
       .spyOn(API, "getTemplateVersionExternalAuth")
       .mockResolvedValueOnce([MockTemplateVersionExternalAuthGithub]);
 
     renderCreateWorkspacePage();
     await waitForLoaderToBeRemoved();
 
-    const nameField = await screen.findByLabelText(nameLabelText);
-
-    // have to use fireEvent b/c userEvent isn't cleaning up properly between tests
```

Comment:
I know that we have the test simplified down now, so it's not an issue anymore, but do you know if we ever figured out the `userEvent` issue? This is the first I've heard of it not cleaning up properly, but some digging made it sounds like it was a bug with the library